ovh/resource_ovh_dns_record.go

```diff
@@ -0,0 +1,183 @@
+package ovh
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type NewRecord struct {
+	Target    string `json:"target"`
+	Ttl       int    `json:"ttl"`
+	FieldType string `json:"fieldType"`
+	SubDomain string `json:"subDomain"`
+}
+
+type Record struct {
+	Id        int    `json:"id"`
+	Zone      string `json:"zone"`
+	Target    string `json:"target"`
+	Ttl       int    `json:"ttl"`
+	FieldType string `json:"fieldType"`
+	SubDomain string `json:"subDomain"`
+}
+
+func resourceOVHDomainZoneRecord() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOVHRecordCreate,
+		Read:   resourceOVHRecordRead,
+		Update: resourceOVHRecordUpdate,
+		Delete: resourceOVHRecordDelete,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"zone": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"target": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"ttl": {
```

Comment:
Is there a reason `ttl` needs to be a `TypeString` and not a `TypeInt`? Would save you from having to call `strconv` during Create.